dokka-integration-tests/gradle/src/testUiShowcaseProject/kotlin/UiShowcaseIntegrationTest.kt

```diff
@@ -24,6 +24,7 @@ class UiShowcaseIntegrationTest : AbstractGradleIntegrationTest(), TestOutputCop
         val result = createGradleRunner(
             buildVersions,
             ":dokkaGenerate",
+            dgpPluginMode = "V2Enabled",
```

Comment:
```suggestion
            dgpPluginMode = null,
```
As the project is configured to use `V2` in `gradle.properties`.